UPGRADING.md

```diff
@@ -51,6 +51,48 @@ class TwitterAPI < Grape::API
 end
 ```
 
+#### Recognizing Path
+
+Due to the changes done in [#2379](https://github.com/ruby-grape/grape/pull/2379), Grape now takes in mind the types of the configured `route_params` in order to determine the endpoint that matches with the performed request. 
```

Comment:
"Grape now considers the types of the configured ..."

"Due to the changes done in ..." -> move to the end of all this text and say "See [xyz] for more information."

Reply:
Thank you so much for the suggestion! I've just applied it! 👏